lib/pavilion/commands/_series.py

```diff
@@ -43,6 +43,6 @@ def run(self, pav_cfg, args):
             series_obj.run()
         except TestSeriesError as err:
             output.fprint(self.errfile, "Error while running series '{}'.".format(args.series_id))
-            output.fprint(self.errfile, err.pformat())
+            output.fprint(self.errfile, err.pformat(args.show_tracebacks))
```

Comment:
This one though, since it's not going to standard out (It's going to a log file) should in fact show the traceback).